tsconfig.base.json

```diff
@@ -22,6 +22,7 @@
 
     /* Strict Type-Checking Options */
     "strict": true /* Enable all strict type-checking options. */,
+    "skipLibCheck": true,
```

Comment:
typescript was having some problems digesting the fact that `@types/eslint` and `@types/eslint-scope` are both in the dependency tree of this package.